contrib/builder/rpm/s390x/generate.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+set -e
+
+# This file is used to auto-generate Dockerfiles for making rpms via 'make rpm'
+#
+# usage: ./generate.sh [versions]
+#    ie: ./generate.sh
+#        to update all Dockerfiles in this directory
+#    or: ./generate.sh centos-7
+#        to only update centos-7/Dockerfile
+#    or: ./generate.sh fedora-newversion
+#        to create a new folder and a Dockerfile within it
+
+cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+versions=( "$@" )
+if [ ${#versions[@]} -eq 0 ]; then
+	versions=( */ )
+fi
+versions=( "${versions[@]%/}" )
+
+for version in "${versions[@]}"; do
+	echo "${versions[@]}"
+	distro="${version%-*}"
+	suite="${version##*-}"
+	case "$distro" in
+		*opensuse*)
+            from="opensuse/s390x:tumbleweed"
```

Comment:
Whitespace throughout this file looks pretty inconsistent -- should be all-tabs.